apps/gauzy/src/app/pages/email-templates/email-templates.component.scss

```
@@ -61,7 +61,11 @@ $height: calc(100vh - 25.5rem);
       border-radius: 0 0 nb-theme(border-radius) nb-theme(border-radius);
 
       .label {
-        background: rgba(231, 227, 218, 0.75);
+        // Was a hardcoded beige (rgba(231, 227, 218, .75)) — a light chip that reads as a
```

Stylelint (17.14.1)

[error] Unknown word Was (CssSyntaxError)

(CssSyntaxError)